document/chapters/01-introduction.tex

```diff
@@ -181,22 +181,26 @@ \section{Statement of Problem}\label{sec:problem-statement}
 confirms that the performance bottleneck of DAGMC lies in its ability to quickly
 satisfy the geometric queries of the underlying Monte Carlo code it is coupled
 to. Looking more closely at the underlying calls in DAGMC, one can see that this
-time is collectively spent in the DAGMC \textit{ray\_fire} method. Function names
-like \textit{RayIntersectSets} and \textit{preorder\_traverse} indicate that
-this time is spent in MOAB's database-oriented ray tracing implementation.
+time is collectively spent in the DAGMC \textit{ray\_fire} method,
+indicating that a significant amount of additional time is spent in
+CAD-based geometry queries.
+
+%% Function names
+%% like \textit{RayIntersectSets} and \textit{preorder\_traverse} indicate that
+%% this time is spent in MOAB's database-oriented ray tracing implementation.
 
 \section{Statement of Thesis}
 
-The purpose of this dissertation is to improve the performance of CAD-based
-Monte Carlo radiation transport in a manner that is widely accessible to
-analysts. The result is a compilation of adaptive data structure construction,
-data structure redesign, and alternative methods to reduce simulation times in
+The purpose of this dissertation is to discover new pathways toward
+improving the performance of CAD-based Monte Carlo radiation transport
```

Comment:
Do you want to change the phrase "CAD-based" here to match the title change?

Reply:
I do. Thank you!